product_docs/docs/pgd/5/rel_notes/index.mdx

```diff
@@ -11,6 +13,8 @@ provide information on what was new in each release. For new functionality
 introduced in a minor or patch release, the content also indicates the release
 that introduced the feature.
 
-| Release Date | EDB Postgres Distributed     | BDR   |
-| ------------ | ---------------------------- | ----- |
-| 2023 Feb 21 | [5.0.0](pgd_5.0.0_rel_notes)  | 5.0.0 |
+| Release Date  |   EDB Postgres Distributed   | BDR extension | PGD CLI | PGD Proxy |  TPA  |
```

Comment:
My only worry is this should not get interpreted as compatibility matrix [like this](https://www.enterprisedb.com/docs/pgd/latest/) because it is not. Also, not sure whether TPA falls under PGD considering it is more general tool (used with other than PGD) and may have more frequent releases than PGD.